src/constants/index.js

```diff
@@ -11,5 +11,7 @@ export const JAVASCRIPT = '/10-days-of-javascript';
 export const CODE = '/30-days-of code'
 export const TEST = '/test'
 export const PROFILE ="/profile"
+export const COURSE_FORM ="/course-form"
```

Comment:
@Humnaa please use proper spacing and consistent convention for your code.

Reply:
I have resolved it and used same convention for whole code